satellite/console/usercredits.go

```diff
@@ -25,15 +23,15 @@ type UserCredit struct {
 	UserID        uuid.UUID
 	OfferID       int
 	ReferredBy    uuid.UUID
-	CreditsEarned rewards.USD
-	CreditsUsed   rewards.USD
+	CreditsEarned int
+	CreditsUsed   int
```

Comment:
Why was this changed to int? This still should be `USD`

Reply:
I mean it would be fine to do it in a separate PR, if you feel like this is getting too large.